package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "to-words",
   "version": "2.4.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@wocodes We have moved on to lockfile version 2. You'd need to either remove package-lock.json from commits or upgrade your npm to ensure correct changes come in.